src/lumigo_tracer/wrappers/http/http_parser.py

```diff
@@ -260,19 +265,33 @@ def get_omit_skip_path() -> Optional[List[str]]:
         return ["PartitionKey"]
 
 
-class SqsParser(ServerlessAWSParser):
+class BaseSqsParser(ServerlessAWSParser, ABC):
```

Comment:
Why did you use inheritance here? I don't see any common code that worths this extra complexity.
Please KISS the tracer (https://he.wikipedia.org/wiki/KISS)

Reply:
The `ServerlessAWSParser` inheritence is required because the `parse_request` & `parse_response` functions call `super().parse_request/parse_response` (the sqs parser adds info about the sqs queue, and the aws parser parses many other properties about the request like amazon request id etc...)

The ABC ingeritence is required because every sqs parser needs to have `_extract_message_id` & `_extract_queue_url` funcs to add information to the request & response in the `parse_request` & `parse_response`, but the implementation for these funcs is different in xml & json sqs parsing.

I can't use the `_extract_message_id` & `_extract_queue_url` functions in the `BaseSqsParser` class without making the class abstract & defining the abstract methods that the children will implement

Reply:
The inheritance itself is useless.
You can do the same work with fewer lines of code and less architectural complexity.
Just remove the Base class

Reply:
Done